lib/jekyll/renderer.rb

```diff
@@ -234,6 +235,14 @@ def assign_highlighter_options!
     end
 
     private
+    def assign_layout_data!
+      layout = layouts[document.data["layout"]]
```

Comment:
Yes, it's the same line as in https://github.com/jekyll/jekyll/blob/master/lib/jekyll/renderer.rb#L145